README.md

```diff
@@ -24,33 +24,10 @@ This repository is organized as follows:
   (displayed when built with `-vv`) will be issued if the build script detects
   a mismatch or is unable to check the installed `libctru` version.
 
-## License
+## Original version
 
-Copyright (C) Rust 3DS Project authors, 2015-2016
```

Comment:
Do we need to retain this copyright header in the LICENSE file? I am not a lawyer but in most cases I've seen you are meant to retain original copyright headers and just add onto them?

Maybe it doesn't apply here since it's a derivative of `ctru-rs-old` rather than a direct copy.

Reply:
Wait, does anyone actually hold a copyright for this name? Don’t you need to file a copyright ©️ for it to have effect?

Reply:
You don't need to file anything to Have the copyright of a creation. The moment you create any work, the copyright is yours